.github/workflows/image-build.yaml

```diff
@@ -175,6 +175,89 @@ jobs:
               # set latest tag for default branch
               type=raw,value=latest,enable={{is_default_branch}}
 
+    name: Build Image (${{ matrix.image.image }})
+    steps:
+      - name: Checkout bpfman
+        uses: actions/checkout@v4
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.5.0
+
+      - name: Generate olm bundle on disk
+        if: ${{ matrix.image.image == 'bpfman-operator-bundle' }}
+        run: |
+          cd bpfman-operator && make bundle
+
+      - name: Login to quay.io/bpfman
+        uses: redhat-actions/podman-login@v1
+        if: ${{ github.event_name == 'push' && matrix.image.repository == 'bpfman'}}
+        with:
+          registry: ${{ matrix.image.registry }}
+          username: ${{ secrets.BPFMAN_USERNAME }}
+          password: ${{ secrets.BPFMAN_ROBOT_TOKEN }}
+
+      - name: Login to quay.io/bpfman-userspace
+        uses: redhat-actions/podman-login@v1
+        if: ${{ github.event_name == 'push' && matrix.image.repository == 'bpfman-userspace'}}
+        with:
+          registry: ${{ matrix.image.registry }}
+          username: ${{ secrets.BPFMAN_USERSPACE_USERNAME }}
+          password: ${{ secrets.BPFMAN_USERSPACE_ROBOT_TOKEN }}
+
+      - name: Login to quay.io/bpfman-bytecode
+        uses: redhat-actions/podman-login@v1
+        if: ${{ github.event_name == 'push' && matrix.image.repository == 'bpfman-bytecode' }}
+        with:
+          registry: ${{ matrix.image.registry }}
+          username: ${{ secrets.BPFMAN_BYTECODE_USERNAME }}
+          password: ${{ secrets.BPFMAN_BYTECODE_ROBOT_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for image
+        id: meta
+        uses: docker/metadata-action@v5.5.1
+        with:
+          images: ${{ matrix.image.registry }}/${{ matrix.image.repository }}/${{ matrix.image.image }}
+          tags: ${{ matrix.image.tags }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and push
+        id: build-push-image
+        uses: docker/build-push-action@v5
+        with:
+          platforms: linux/amd64, linux/arm64
+          # platforms: linux/amd64, linux/arm64, linux/ppc64le, linux/s390x
```

Comment:
nit remove this comment maybe just add one saying "TBD linux/ppc64le, linux/s390x"

Reply:
Done